collection-scripts/gather

```diff
@@ -28,3 +29,31 @@ resources+=(clusterautoscaler machineautoscaler)
 for resource in ${resources[@]}; do
     /usr/bin/openshift-must-gather inspect ${resource}
 done
+
+# Collect System Audit Logs
+AUDIT_LOG_PATH="${BASE_COLLECTION_PATH}/audit_logs/"
+mkdir -p ${AUDIT_LOG_PATH}
+/usr/bin/oc adm node-logs --role=master --path=openshift-apiserver/ > ${AUDIT_LOG_PATH}/openshift-apiserver_audit.log
```

Comment:
This handles the fact that we gzip the files?

Reply:
@deads2k IDK what you're trying to ask here.

Reply:
I'll merge it and we'll see if we get valid audit logs out the other end.

Reply:
This is wrong, when `--path` is directory you'll only get listing of that dir, only when it's a actual file it will get downloaded, compare https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml#L439-L444 how collect audit logs in our CI.

Reply:
See:
```
$ oc adm node-logs --role=master --path=openshift-apiserver/
ip-10-0-133-247.us-east-2.compute.internal audit.log
ip-10-0-150-168.us-east-2.compute.internal audit.log
ip-10-0-167-24.us-east-2.compute.internal audit.log
```

Reply:
I pulled what we have in CI into this. @soltysh PTAL